action.yml

```diff
@@ -1,18 +1,31 @@
-name: Trigger Management Function
-description: Trigger a management function in a new changeset
-author: jkeiser@systeminit.com
-
-# Add your action's branding here. This will appear on the GitHub Marketplace.
+name: System Initiative GitHub Action
+description:
+  Interact with the System Initiative workspace from within a GitHub Action
+author: System Initiative
 branding:
-  icon: heart
-  color: red
+  icon: box
+  color: black
 
 # Define your inputs here.
 inputs:
-  changeSetName:
-    description: The name of the changeset to create.
-
-      By default this is required, unless changeSetId is passed.
+  workspaceId:
```

Comment:
Leaning into your intuition that these are going to confuse people most of the time (until they really need them), I think we should stuff these down further in the yaml (if we leave them here at all) and put some kind of big marker on them denoting how ridiculously optional they are. Putting them at the top will make some people puzzle through and try to type them. (I actually think what you did with viewId and managementPrototypeId--removing them entirely--might be the right thing.)

And for `changeSet`, I think we shouldn't reference changeSetId honestly. Let's leave the "Required ..." stuff in changeSetId only and leave this one at the top of the docs.